src/FrontendObligationChecker/ViewModels/AmountYouSupply.cs

```diff
@@ -40,30 +40,32 @@ private static List<Question> GetNonSellerActivityYesQuestions(Page page)
     private static bool CheckHasSingleUseDrinkContainers(Page page)
     {
         var isPlacingDrinksOnMarket =
-            page.FindPage(PagePath.PlaceDrinksOnMarket)?.Questions.FirstOrDefault().Answer == YesNo.Yes;
+        page.FindPage(PagePath.PlaceDrinksOnMarket)?.Questions.FirstOrDefault()?.Answer != null
+            && page.FindPage(PagePath.PlaceDrinksOnMarket)?.Questions.FirstOrDefault()?.Answer == YesNo.Yes;
 
         if (!isPlacingDrinksOnMarket)
         {
             return false;
         }
 
         bool isUsingDrinkContainersMaterials =
-            page.FindPage(PagePath.MaterialsForDrinksContainers)?.Questions.FirstOrDefault().Answer != "0";
+        page.FindPage(PagePath.MaterialsForDrinksContainers)?.Questions.FirstOrDefault()?.Answer != null
+              && page.FindPage(PagePath.MaterialsForDrinksContainers)?.Questions.FirstOrDefault()?.Answer != "0";
 
         if (!isUsingDrinkContainersMaterials)
         {
             return false;
         }
 
-        var containerVolumePage = page.FindPage(PagePath.ContainerVolume);
-        return containerVolumePage?.Questions.FirstOrDefault().Answer == YesNo.Yes;
+        return page.FindPage(PagePath.ContainerVolume)?.Questions.FirstOrDefault()?.Answer != null
```

Comment:
Please add a unit test, not sure why you are calling 
page.FindPage(PagePath.ContainerVolume) and Questions.FirstOrDefault()?.Answer twice